core/src/test/java/org/infinispan/xsite/backupfailure/tx/BaseBackupTxFailureTest.java

```diff
@@ -33,11 +32,10 @@ protected void createSites() {
       backup("LON").getAdvancedCache().getAsyncInterceptorChain().addInterceptor(failureInterceptor, 1);
    }
 
-   @Test(groups = {"xsite", "unstable"})
    public void testPrepareFailure() {
       failureInterceptor.enable();
       try {
-         Exceptions.expectException(XAException.class, () -> cache("LON", 0).put("k", "v"));
+         Exceptions.expectException(CacheException.class, () -> cache("LON", 0).put("k", "v"));
```

Comment:
The `XaException` are never seen by the user. they are only there for interaction between the `TransactionManager` and the `XaResource`/`Synchronization`. 
I'll change to check for `RollbackException`